assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -143,6 +139,7 @@ function DisplayElement({ element, previewMode }) {
 DisplayElement.propTypes = {
   previewMode: PropTypes.bool,
   element: StoryPropTypes.element.isRequired,
+  page: StoryPropTypes.page,
```

Comment:
`isRequired`. Also this might fail some tests, so feel free to plug them as needed.

Reply:
I already checked the tests. Making the prop types required is good, but the component breaks if it's null anyway.